portfolio/src/components/Connect.vue

```diff
@@ -1,15 +1,116 @@
+            <div id="message-input-container" class="comment">
+                <v-text-field 
+                    style="width: 100%" 
+                    label="Name"
+                    v-model="name"
+                    color="#DBDBDB"
+                    dark
+                ></v-text-field>
+                <v-textarea
+                    v-model="message"
+                    label="Message"
+                    style="width: 100%" 
+                    color="#DBDBDB"
+                    dark
+                ></v-textarea>
+                <v-btn
+                    v-on:click="sendComment()"
+                    text
+                    style="margin: 0.5rem 0 0 auto;"
+                >Send</v-btn>
+            </div>
+
+            <div id="comment-list">
+                <div 
+                    class="comment"
+                    v-for="(comment, index) in commentList"
+                    :key="index"
+                >
+                    <div class="comment-header-container">
+                        <h1 style="font-size: 18px">{{comment.name}}</h1>
+                        <span style="color: #a3a2a2; font-size: 14px">{{comment.timeCreated}}</span>
+                    </div>
+                    <p style="width: 100%">{{comment.message}}</p>
+
+                </div>
+            </div>
+        </div>
     </div>
 </template>
 
 <script>
 export default {
-  name: 'Connect',
-  props: {
-    msg: String
-  }
+    name: 'Connect',
+
+    created () {
+        this.getDataFromServer().then(function () {
+            console.log("Finished!")
+        })
+    },
+    
+    data () {
+        return {
+            name: "",
+            message: "",
+            commentList: []
+        }
+    },
+
+    methods: {
+        sendComment: async function() {
+
+            const userName = this.name
+            const userMessage = this.message
+            let response = await fetch('https://8080-dot-12454009-dot-devshell.appspot.com/comments', {
+                method: "POST",
+                headers: {
+                    "Content-Type": "application/json"
+                },
+                body: JSON.stringify({
+                    datePosted: null,
+                    name: userName,
+                    message: userMessage
+                })
+            })
+
+            if (response.ok) {
+                console.log("POST OK")
+                this.name = ""
+                this.message = ""
+            } else {
+                console.log("POST ERR")  
+            }
+
+
+            // Call the get command to update the UI with the comment sent via the POST above
+            await this.getDataFromServer()
+
+        },
+
+        getDataFromServer: async function () {
+            let response = await fetch('https://8080-dot-12454009-dot-devshell.appspot.com/comments')
+
+            if (response.ok) {
+                let respData = await response.json()
+                console.log(respData)
```

Comment:
Is it for debug? ideally in real life, we don't want to print these data in the console?

Reply:
Yep, I was having this weird issue where the data was coming back in a different format and forgot to remove, thank you!